socs/agents/rtsp_camera/agent.py

```diff
@@ -177,6 +177,18 @@ def _dt_convert(timestr):
         else:
             return False
 
+    def _init_stream(self):
+        """Connect to camera and initialize video stream."""
+        if self.fake:
+            cap = FakeCamera()
+        else:
+            cap = cv2.VideoCapture(self.connection)
+        if not cap.isOpened():
+            self.log.error(f"Cannot open RTSP stream at {self.connection}")
+            return False, "Could not open RTSP stream"
```

Comment:
This is returning a tuple if `cap` doesn't open, which results in `not cap:` evaluating to `False` down on line 226, so this will always result in `connected = True`. It made sense to return this tuple within the process, since that's what ocs needs, but now that it's moved best to just return `None`, that will evaluate the connected check correctly.

```python
>>> cap = False, "Could not open RTSP stream"
>>> connected = True
>>> if not cap:
...     connected = False
...
>>> print(connected)
True
```